boilerplate/fullstack/remix/app/lib/superTokensHelpers.ts

```diff
@@ -139,3 +146,147 @@ export async function getSessionDetails(
         }
     }
 }
+
+async function commonSSRSession(
```

Comment:
the getSessionDetails function is essentially equal to this one. So remove one of them please.